domain/src/main/kotlin/com/studentcenter/weave/domain/entity/DeactivateUserInfo.kt

```diff
@@ -0,0 +1,33 @@
+package com.studentcenter.weave.domain.entity
+
+import com.studentcenter.weave.domain.enum.SocialLoginProvider
+import com.studentcenter.weave.support.common.uuid.UuidCreator
+import com.studentcenter.weave.support.common.vo.Email
+import java.time.LocalDateTime
+import java.util.UUID
+
+data class DeactivateUserInfo(
+    val id: UUID = UuidCreator.create(),
+    val email: Email,
+    val socialLoginProvider: SocialLoginProvider,
+    val reason: String? = null,
+    val registeredAt: LocalDateTime,
+    val deactivatedAt: LocalDateTime = LocalDateTime.now(),
+) {
+
+    companion object {
+        fun create(
+            user: User,
+            userAuthInfo: UserAuthInfo,
+            reason: String?,
```

Comment:
p4) 생성자에도 같이 들어가면 좋을 것 같아요!
```suggestion
            reason: String? = null,
```